src/scripts/browserDeviceProfile.js

```diff
@@ -670,18 +696,16 @@ export default function (options) {
         if (browser.safari) {
             hlsInFmp4VideoCodecs.push('vp9');
         }
-        // webm support is unreliable on safari 17
-        if (!browser.safari
-             || (browser.safari && browser.versionMajor >= 15 && browser.versionMajor < 17)) {
+        // WebM support is unreliable in some versions of Safari.
+        if (!browser.safari || (browser.safari && safariSupportsWebm)) {
```

Comment:
You can omit the second `browser.safari` because `!X || (X && Y) = (!X || X) && (!X || Y)` and `!X || X` is always `true`.
```suggestion
        if (!browser.safari || safariSupportsWebm) {
```